test/e2e/e2e_operator_test.go

```diff
@@ -84,24 +76,20 @@ func findKueuePods(list *corev1.PodList) []*corev1.Pod {
 }
 
 var _ = Describe("Kueue Operator", Ordered, func() {
-	var (
-		namespace = operatorNamespace
-	)
 	// AfterEach(func() {
 	// 	Expect(kubeClient.CoreV1().Namespaces().Delete(context.TODO(), namespace, metav1.DeleteOptions{})).To(Succeed())
 	// })
 	When("installs", func() {
 		It("operator pods should be ready", func() {
-			Expect(deployOperator()).To(Succeed(), "operator deployment should not fail")
```

Comment:
Why did you remove this?

Reply:
I just moved it to line 104. This function is deploying the operand, so I think it makes sense to add it to the test step that checks if it's running, instead of calling it in the step that checks if the operator pods are up.